.idea/vcs.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is editor-specific. Please remove it and add `.idea/` to the `.gitignore` file

Reply:
done